homeassistant/components/unifi/unifi_client.py

```diff
@@ -72,6 +74,7 @@ def async_update_callback(self) -> None:
                 self.wireless_connection = self.client.event.event in (
                     WIRELESS_CLIENT_CONNECTED,
                     WIRELESS_CLIENT_ROAM,
+                    WIRELESS_CLIENT_ROAMRADIO,
```

Comment:
Same question about if guest events should be included here?

Reply:
See previous answer